### PR TITLE
Finance report revenue by client

### PR DESCRIPTION
**Pull request by yuvraj752**
Targets #2853 

### Description

These changes are required to add landing page with 2 cards revenue visualization and client revenue report in finance -> report ->revenue by client.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by yuvraj752** [comment]
I have created a landing page for Revenue by Client.

Capture

clicking revenue visualization card opens client revenue graph and the client revenue report card is disabled.

Capture1

Capture2

@pankaj-ag if there is any changes thats need to be made feel free to let me know.

**Comment by yuvraj752** [comment]
I have created a landing page for Revenue by Client.

Capture

clicking revenue visualization card opens client revenue graph and the client revenue report card is disabled.

Capture1

Capture2

@pankaj-ag if there is any changes thats need to be made feel free to let me know.

File: Modules/Report/Http/Controllers/FinanceReportController.php [hunk 1]
```diff
@@ -40,4 +40,9 @@ public function getReportData(Request $request)
 
         return $this->reportDataService->getData($type, json_decode($filters, true));
     }
+
+    public function index()
+    {
+        return view('report::finance.client-wise-revenue.dashboard');
+    }
```
Comment (pankaj-ag) [inline Modules/Report/Http/Controllers/FinanceReportController.php:47]: As per the standards let's keep the index function as the first function of the controller

**Review by pankaj-ag — APPROVED** [review]
Please fix it and then we can merge.

File: Modules/Report/Http/Controllers/FinanceReportController.php (hunk 1 above)
Comment (yuvraj752) [reply Modules/Report/Http/Controllers/FinanceReportController.php]: understood

**Pull request merged**